.github/workflows/ci.yml

```diff
@@ -17,7 +17,18 @@ jobs:
         node-version: ${{ matrix.node-version }}
 
     - name: Install dependencies
-      run: npm ci
+      run: |
+        npm ci
+        sudo apt install fonts-noto-color-emoji
```

Comment:
What's this for?

Reply:
Providing unicode (non @readme emojis) support.

Ideally, we'd run the browser test in a docker container, so we can duplicate the env that ci uses.